app/models/communication_log.rb

```diff
@@ -18,4 +18,8 @@ def self.log_submission_email(email_object, delivery_status, submission, deliver
                  created_by: current_user
     )
   end
+
```

Comment:
@exbinary you'd recommended moving presenter methods elsewhere. would love your suggestions!